pkg/querier/tripperware/queryrange/query_range.go

```diff
@@ -230,7 +230,7 @@ func (c prometheusCodec) DecodeResponse(ctx context.Context, r *http.Response, _
 	return &resp, nil
 }
 
-func (prometheusCodec) EncodeResponse(ctx context.Context, res tripperware.Response) (*http.Response, error) {
+func (prometheusCodec) EncodeResponse(ctx context.Context, _ *http.Request, res tripperware.Response) (*http.Response, error) {
```

Comment:
Seems weird that we need to pass in a request here if it's not being used.

Reply:
It is just for the `interface`.
It is not used in the `range query`, since the ruler only does `instant query`,